src/gfx/geometries/ChunkedObjectsGeometry.js

```diff
@@ -5,141 +5,140 @@ import * as THREE from 'three';
 import utils from '../../utils';
```

Comment:
There are extra blank lines at the top of several files (they were inserted by mistake automatically long time ago). Please remove them all during your refactoring so that we don't need to fix this in a separate PR. Also ensure that a single (and the only) blank line is present at the bottom of each file. Thank you.